src/main/groovy/com/twcable/grabbit/server/batch/steps/jcrnodes/JcrNodesProcessor.groovy

```diff
@@ -59,152 +49,31 @@ class JcrNodesProcessor implements ItemProcessor<JcrNode, ProtoNode> {
     @Nullable
     ProtoNode process(JcrNode jcrNode) throws Exception {
 
+        JCRNodeDecorator decoratedNode = new JCRNodeDecorator(jcrNode);
+
         //TODO: Access Control Lists nodes are not supported right now.
-        if (!jcrNode || jcrNode.path.contains("rep:policy")) {
-            log.info "Ignoring current node ${jcrNode}"
+        if (!decoratedNode || decoratedNode.path.contains("rep:policy")) {
+            log.info "Ignoring current node ${decoratedNode.innerNode}"
```

Comment:
Minor, but this shouldn't be necessary since decoratedNode delegates toString() to the innerNode anyway